pickee_api/models.py

```diff
@@ -1,3 +1,65 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MaxValueValidator
 
-# Create your models here.
+class UserProfile(models.Model):
+    user = models.OneToOneField(User)
+    user.first_name.blank = False
+    user.last_name.blank = False
+
+    picture = models.ImageField(upload_to='profile_images', blank=True)
+
+    GENDER_CHOICES = ('Male', 'Female', 'Other', 'Prefer not to say')
+    gender = models.CharField(max_length=18, choices=GENDER_CHOICES)
+    age = models.IntegerField(validators=[MaxValueValidator(100), MinValueValidator(0)])
+    associated_users = models.ManyToManyField('self')
+
+    def __str__(self):
+        return self.user.first_name + " " + self.user.last_name
+
+class FavouriteActors(models.Model):
+    user = models.ForeignKey('UserProfile', on_delete=models.CASCADE)
+    name = models.CharField(max_length=128, unique=True)
```

Comment:
My understanding is that we can have the same actors multiple times.
Like this:
```
User, Name
1, Pedro Pascal
1, Brad Pitt
2, Pedro Pascal
2, Margot Robbie
3, Leonardo Di Caprio
```